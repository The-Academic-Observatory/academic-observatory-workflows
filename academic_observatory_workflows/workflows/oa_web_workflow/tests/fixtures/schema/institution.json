version https://git-lfs.github.com/spec/v1
oid sha256:95116fce1d624a322c070e383b301f3612b43639eea02aa8381bb07664b51ca6
size 42279
