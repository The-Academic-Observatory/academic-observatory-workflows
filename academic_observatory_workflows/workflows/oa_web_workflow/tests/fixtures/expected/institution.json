version https://git-lfs.github.com/spec/v1
oid sha256:9e5d3c908331645314d8d950fd493e8e8d7a97f9802123ed4675997aa37b9174
size 83575
